### DISCUSSION TITLE
Kotlin 协程与 Retrofit

### DISCUSSION THREAD

**Discussion by yujinyan** [yujinyan/blog#174] [Comments]
https://blog.yujinyan.me/posts/kotlin-coroutine-retrofit/

**Comment by omyrobin** [comment]
写的真是太棒了

**Comment by mariohide** [comment]
非常感谢作者，很优质的文章！

**Comment by oxyroid** [comment]
请问作者这种blog样式是怎么做的，需要会react吗？

**Comment by yujinyan** [comment]
> 请问作者这种blog样式是怎么做的，需要会react吗？

博客是在 [gatsby-starter-blog](https://www.gatsbyjs.com/starters/gatsbyjs/gatsby-starter-blog) 这个模版的基础上做了些开发，是基于 react 的。代码在 https://github.com/yujinyan/blog 可以参考～

**Comment by lonelee** [comment]
学习了很多，感谢作者

**Comment by SarahSmitho** [comment]
大佬你还有其他文章在其他平台吗？写得好好，看完你的文章意犹未尽，还想再看更多

**Comment by mariohide** [comment]
作者您好，`kotlinx.serialization` 现在已经支持流式解析了，您是否有兴趣推出 `kotlinx.serialization` 版本的「拆解信封」的案例教学？

**Reply by mai2579017147** [reply]
期待期待

**Reply by mai2579017147** [reply]
收获很大，感谢！会出kotlinx.serialization的教学吗
